README.asciidoc

```diff
@@ -134,7 +134,7 @@ brew install --HEAD https://raw.githubusercontent.com/mawww/kakoune/master/contr
 
 To update kakoune,
 -----------------------------------------------------------------------------------------------
-brew reinstall --HEAD https://raw.githubusercontent.com/mawww/kakoune/master/contrib/kakoune.rb
+brew upgrade --fetch-HEAD kakoune
 -----------------------------------------------------------------------------------------------
```

Comment:
Done. I didn't touch the lines on the other code blocks because I didn't want to conflict with #980.